src/Components/Pages/SuccessPage.js

```diff
@@ -27,16 +27,25 @@ const SuccessPage = ({ isLoading, name, onClose }) => {
         headingLevel="h4"
       />
       <EmptyStateBody>
-        <b>{name}</b> is now available for use. Click <b>View activation key</b>{' '}
-        to edit settings or add repositories.
+        <b>{name}</b> is now available for use.{' '}
+        {isRegAssistant ? (
+          'Close to continue with the registration assistant'
+        ) : (
+          <span>
+            Click <b>View activation key</b> to edit settings or add
+            repositories.
+          </span>
+        )}
```

Comment:
I would prefer if we could avoid mixing in business logic here. The success page, and even the whole modal, shouldn't really need to know what context its been opened in.

Maybe we could inject the success page via a prop to the create wizard, and then the registration assistant could inject its own custom success page (and then have control over any changes to it as well) and we could inject the current success page

Reply:
Or maybe even make the prop optional - if an alternative success page is provided, render it, otherwise use the default one